ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -109,12 +109,15 @@ public synchronized boolean setReplicaSealedState(ReplicaId replicaId, boolean i
     String partitionId = replicaId.getPartitionId().toPathString();
     boolean success = true;
     if (!isSealed && sealedReplicas.contains(partitionId)) {
+      logger.trace("Removing the partition {} from sealReplicas list", partitionId);
       sealedReplicas.remove(partitionId);
       success = setSealedReplicas(sealedReplicas);
     } else if (isSealed && !sealedReplicas.contains(partitionId)) {
+      logger.trace("Adding the partition {} from sealReplicas list", partitionId);
```

Comment:
minor: `Adding the partition {} to sealReplicas list`

Reply:
Got it. Will address this in next PR. Thanks!